messages_appendix.cddl

```diff
@@ -266,13 +266,19 @@ remote-playback-availability-event = {
 remote-playback-start-request = {
   request
   1: remote-playback-id ; remote-playback-id
-  2: [* text] ; urls
+  2: [* remote-playback-source] ; sources
   ? 3: [* text] ; text-track-urls
   ? 4: [* http-header] ; headers
   ? 5: remote-playback-controls ; controls
   ? 6: {streaming-session-start-request-params} ; remoting
 }
 
+remote-playback-source = {
+  1: text; url
+  2: text; css-media-query
```

Comment:
.media and .type are both optional for HtmlMediaElement, right?
So the message fields should also be optional I think.

Reply:
My reading of HTML is that the media query is only applicable if the `<source>` element is being used to select an image file in a `<picture>`.  So this can be dropped.

https://html.spec.whatwg.org/multipage/embedded-content.html#the-source-element

Reply:
I think it's okay to make extended-mime-type mandatory; per the WebIDL, an omitted attribute gets reflected to script as an empty string.

Reply:
I updated the PR to reflect this.